timemachine/fe/system.py

```diff
@@ -129,29 +134,22 @@ def get_U_fn(self):
         """
         Return a jax function that evaluates the potential energy of a set of coordinates.
         """
+        assert self.torsion
+        U_fns = self.get_U_fns()
 
         def U_fn(x):
-            assert self.torsion
-
-            # Chiral restraints are disabled until checks are added (see GH #815)
-            # chiral_U = chiral_atom_U(x) + chiral_bond_U(x)
-            return (
-                self.bond(x, box=None)
-                + self.angle(x, box=None)
-                + self.torsion(x, box=None)
-                + self.nonbonded(x, box=None)
-                # + self.chiral_atom(x, box=None)
-                # + self.chiral_bond(x, box=None)
-            )
+            return sum(U(x, box=None) for U in U_fns)
```

Comment:
Should this be `jnp.sum()`? It looks like jax handles the builtin `sum` correctly, so maybe not.

Reply:
You're right.

(When I wrote this, wasn't sure whether every instance of `BoundPotential` returns a jax array or if it may return a float. (Just confirmed these would never be `BoundGpuImplWrapper`, which returns float.))